diff --git a/Source/Svg.csproj b/Source/Svg.csproj
--- a/Source/Svg.csproj
+++ b/Source/Svg.csproj
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>net8.0;net9.0;netcoreapp3.1;netstandard2.1;netstandard2.0;net462;net472;net481</TargetFrameworks>
+        <TargetFrameworks>net10.0;net8.0;net9.0;netcoreapp3.1;netstandard2.1;netstandard2.0;net462;net472;net481</TargetFrameworks>
         <PackageLicenseExpression>MS-PL</PackageLicenseExpression>
         <RootNamespace>Svg</RootNamespace>
         <AssemblyName>Svg</AssemblyName>
@@ -71,4 +71,5 @@
         <DefineConstants Condition="$(TargetFramework.StartsWith('netcore'))">$(DefineConstants);NETCORE</DefineConstants>
         <DefineConstants Condition="$(TargetFramework.StartsWith('net8'))">$(DefineConstants);NETCORE;NETNEXT</DefineConstants>
         <DefineConstants Condition="$(TargetFramework.StartsWith('net9'))">$(DefineConstants);NETCORE;NETNEXT</DefineConstants>
+        <DefineConstants Condition="$(TargetFramework.StartsWith('net10'))">$(DefineConstants);NETCORE;NETNEXT</DefineConstants>
     </PropertyGroup>
@@ -151,2 +152,6 @@
         <PackageReference Include="System.Drawing.Common" Version="5.0.3" />
     </ItemGroup>
+
+    <ItemGroup Condition="'$(TargetFramework)' == 'net10.0' and '$(UseProGpuSystemDrawing)' == 'true'">
+        <ProjectReference Include="$(ProGpuSourceRoot)/src/System.Drawing.Common/System.Drawing.Common.csproj" />
+    </ItemGroup>
